<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>420</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Exam Seating</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>120</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>590</x>
             <y>50</y>
             <width>171</width>
             <height>20</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;; color: blue</string>
           </property>
           <property name="text">
            <string>Scan Student Card</string>
           </property>
          </widget>
          <widget class="QLabel" name="seatRFIDIcon">
           <property name="geometry">
            <rect>
             <x>440</x>
             <y>10</y>
             <width>100</width>
             <height>100</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>50</y>
             <width>111</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Student Name:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>20</y>
             <width>91</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Student No:</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="seatLEStudentName">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>50</y>
             <width>181</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="seatLEStudentNo">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>20</y>
             <width>181</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="staticMessageExamSeating">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>80</y>
             <width>311</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="seatTableWidget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <column>
           <property name="text">
            <string>Date</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Module</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Venue</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Seat</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Toilet Trips</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>120</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <widget class="QLabel" name="toiletScanLabel">
           <property name="geometry">
            <rect>
             <x>590</x>
             <y>50</y>
             <width>171</width>
             <height>20</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;; color: blue</string>
           </property>
           <property name="text">
            <string>Scan Student Card</string>
           </property>
          </widget>
          <widget class="QLabel" name="toiletRFIDIcon">
           <property name="geometry">
            <rect>
             <x>440</x>
             <y>10</y>
             <width>100</width>
             <height>100</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_15">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>30</y>
             <width>111</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Student Name:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_16">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>0</y>
             <width>91</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Student No:</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="toiletLEStudentName">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>30</y>
             <width>181</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="toiletLEStudentNo">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>0</y>
             <width>181</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_17">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>60</y>
             <width>91</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Lecturer ID :</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="toiletLELecturerID">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>60</y>
             <width>181</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="staticMessageToiletTrips">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>90</y>
             <width>441</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="toiletTableWidget">
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Early Submission</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>120</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <widget class="QLabel" name="label_18">
           <property name="geometry">
            <rect>
             <x>590</x>
             <y>50</y>
             <width>171</width>
             <height>20</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;; color: blue</string>
           </property>
           <property name="text">
            <string>Scan Student Card</string>
           </property>
          </widget>
          <widget class="QLabel" name="subRFIDIcon">
           <property name="geometry">
            <rect>
             <x>440</x>
             <y>10</y>
             <width>100</width>
             <height>100</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_20">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>50</y>
             <width>111</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Student Name:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_21">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>20</y>
             <width>91</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Student No:</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="subLEStudentName">
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>50</y>
             <width>181</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="subLEStudentNo">
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>20</y>
             <width>181</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="staticMessageSubmission">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>90</y>
             <width>401</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="subTableWidget">
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
